deploy/openshift/templates/che-server-template.yaml

```diff
@@ -142,6 +142,8 @@ objects:
                 key: ca.crt
                 name: openshift-identity-provider
                 optional: true
+          - name: CHE_WORKSPACE_FEATURE_API
+            value: "http://feature-api-service:3000"
```

Comment:
Does it mean that WS next functionality will be enabled even when feature API is not deployed?

Reply:
Yes, it will be configured but it won't work unless someone adds features property in his workspace. So, I think it is OK to set it by default to simplify deployment in case Workspace Next functionality is needed.